### PR TITLE
adding unit test to jsonfile reader

### PR DESCRIPTION
**Pull request by LimbertVargas**

### CODE REVIEW

File: test/test_RequestApi.py [hunk 1]
```diff
@@ -44,3 +46,12 @@ def test_replace_variable_sends_invalid_dictionary_and_input_endpoint_return_url
...
     assert expected == actual
+
+
+def test_read_file_path_and_return_new_file_path():
+    file_path = "trello/jsonfiles/file.json"
+    new_file_path = path.realpath(file_path)
+    expected = {"name": "limbert"}
+    json_reader = JsonFileReader()
+    actual = json_reader.read(file_path)
+    assert expected == actual
```
Comment (JuanCarlosMartinez4) [inline test/test_RequestApi.py]: add a new line at end of code lines

File: trello/jsonfiles/file.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "name": "limbert"
+}
```
Comment (JuanCarlosMartinez4) [inline trello/jsonfiles/file.json]: add a new line at end of code lines

File: test/test_RequestApi.py (hunk 1 above)
Comment (JuanCarlosMartinez4) [inline test/test_RequestApi.py]: what is the objective of this unit test?

File: test/test_RequestApi.py [hunk 3]
```diff
@@ -44,3 +46,12 @@ def test_replace_variable_sends_invalid_dictionary_and_input_endpoint_return_url
...
     actual = request_api.replace_variables(endpoint, id_dictionary)
     assert expected == actual
+
+
+def test_read_file_path_and_return_new_file_path():
+    file_path = "trello/jsonfiles/file.json"
+    new_file_path = path.realpath(file_path)
+    expected = {"name": "limbert"}
+    json_reader = JsonFileReader()
+    actual = json_reader.read(file_path)
```
Comment (Enrique-C) [inline test/test_RequestApi.py]: Review this, that class is static

File: test/test_RequestApi.py (hunk 1 above)
Comment (LimbertVargas) [reply test/test_RequestApi.py]: done

File: trello/jsonfiles/file.json (hunk 2 above)
Comment (LimbertVargas) [reply trello/jsonfiles/file.json]: done

File: test/test_RequestApi.py (hunk 1 above)
Comment (LimbertVargas) [reply test/test_RequestApi.py]: return the new file path in dictionary

File: test/test_RequestApi.py (hunk 3 above)
Comment (LimbertVargas) [reply test/test_RequestApi.py]: done

File: test/test_RequestApi.py [hunk 4]
```diff
@@ -39,3 +42,11 @@ def test_replace_variable_sends_invalid_dictionary_and_input_endpoint_return_url
     expected = "/boards/boardObject.id/labels"
     actual = RequestApiUtils.replace_variables(endpoint, id_dictionary)
     assert expected == actual
+
+
+def test_read_file_path_and_return_new_file_path():
+    file_path = "trello/jsonfiles/file.json"
+    new_file_path = path.realpath(file_path)
+    expected = {"name": "limbert"}
```
Comment (JuanCarlosMartinez4) [inline test/test_RequestApi.py]: change the name for testing

File: trello/jsonfiles/file.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "name": "limbert"
```
Comment (JuanCarlosMartinez4) [inline trello/jsonfiles/file.json]: change the name for testing

File: test/test_RequestApi.py [hunk 6]
```diff
@@ -39,3 +42,11 @@ def test_replace_variable_sends_invalid_dictionary_and_input_endpoint_return_url
     expected = "/boards/boardObject.id/labels"
     actual = RequestApiUtils.replace_variables(endpoint, id_dictionary)
     assert expected == actual
+
+
+def test_read_file_path_and_return_new_file_path():
```
Comment (JuanCarlosMartinez4) [inline test/test_RequestApi.py]: update the name

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Review by oscarlopez8 — APPROVED** [review]

**Pull request merged**